include/translated-object.php

```diff
@@ -253,10 +253,32 @@ public function get_translations( $id ) {
 			return array();
 		}
 
+		$translations = $this->get_raw_translations( $id );
+
+		return $this->validate_translations( $translations, $id, 'display' );
+	}
+
+	/**
+	 * Returns an array of translations of an object.
```

Comment:
```suggestion
	 * Returns an unvalidated array of translations of an object.
	 * It is generally preferable to use `get_translations()`.	 
```

Reply:
Done in 5b7a05c610daf5577e514cab5da5644ce9e93cac